### PR TITLE
Juniper contains a unnecessary packet size at the end of the message when used with TCP

### PR DESCRIPTION
**Pull request by philippkahr**
## What does this PR do?

I appended the existing grok pattern for a `(?<_temp_.optionalnumber>\d+)?$'` which should take care of the additional packet size (at least that is what I think it is). I don't think that there is any valid information in that, so we can just drop it, that is why I am parking the value inside the `_temp_` object.

A normal juniper message send by UDP looks like this:

```
<12>1 2016-02-18T01:32:50.391Z utm-srx550-b RT_UTM - WEBFILTER_URL_BLOCKED [junos@67.43.156.15 source-address="192.168.1.100" source-port="58071" destination-address="67.43.156.13" destination-port="80" category="cat1" reason="BY_BLACK_LIST" profile="uf1" url="www.baidu.com" obj="/" username="user01" roles="N/A"]
```

However, when send per TCP it is this:

```
<12>1 2016-02-18T01:32:50.391Z utm-srx550-b RT_UTM - WEBFILTER_URL_BLOCKED [junos@67.43.156.15 source-address="192.168.1.100" source-port="58071" destination-address="67.43.156.13" destination-port="80" category="cat1" reason="BY_BLACK_LIST" profile="uf1" url="www.baidu.com" obj="/" username="user01" roles="N/A"]1002
```

There is an appended `1002` any sort of number attached after the last `]`. This does not work with the existing grok pattern.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/juniper_srx/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.10.1-next"
+  changes:
+    - description: Juniper attaches packet size to tcp input
```
Comment (efd6) [inline packages/juniper_srx/changelog.yml:4]: Can you make this a description of what the change does?

```suggestion
    - description: Handle Juniper packet size suffix in TCP input.
```

File: packages/juniper_srx/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ processors:
   - grok:
       field: event.original
       patterns:
-      - '^<%{POSINT:syslog_pri}>(\d{1,3}\s)?(?:%{TIMESTAMP_ISO8601:_temp_.raw_date})\s%{SYSLOGHOST:syslog_hostname}\s%{PROG:syslog_program}\s(?:%{POSINT:syslog_pid}|-)?\s%{WORD:log_type}\s\[([^=]+?\s)?%{GREEDYDATA:_temp_.original}\]$'
+      - '^<%{POSINT:syslog_pri}>(\d{1,3}\s)?(?:%{TIMESTAMP_ISO8601:_temp_.raw_date})\s%{SYSLOGHOST:syslog_hostname}\s%{PROG:syslog_program}\s(?:%{POSINT:syslog_pid}|-)?\s%{WORD:log_type}\s\[.+?\s%{GREEDYDATA:_temp_.original}\](?<_temp_.optionalnumber>\d+)?$'
```
Comment (efd6) [inline packages/juniper_srx/data_stream/log/elasticsearch/ingest_pipeline/default.yml:16]: Why is the `[^=]` relaxed to a `.`? This seems unrelated.

**Review by efd6 — COMMENTED** [review]
Is there a reference for this behaviour?

File: packages/juniper_srx/data_stream/log/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (philippkahr) [reply packages/juniper_srx/data_stream/log/elasticsearch/ingest_pipeline/default.yml:16]: Oh yes, you are right, that is an oversight by me.
